package.json

```diff
@@ -13,7 +13,7 @@
     "tdd": "mocha --watch",
     "test": "npm run lint && npm run test:once && npm run test:types",
     "test:once": "nyc --reporter=lcovonly --reporter=html  mocha",
-    "test:types": "dtslint --expectOnly --localTs node_modules/typescript/lib types",
```

Comment:
this didn't work before I made changes, kept returning

```> schema-typed@2.0.3 test:types
> dtslint --expectOnly --localTs node_modules/typescript/lib types

Error: Errors in typescript@local for external dependencies:
../src/Schema.ts(33,37): error TS2345: Argument of type 'DataType' is not assignable to parameter of type 'PlainObject<any>'.
../src/Schema.ts(48,37): error TS2345: Argument of type 'DataType' is not assignable to parameter of type 'PlainObject<any>'.

    at /Users/jspears/dev/schema-typed/node_modules/dtslint/bin/index.js:207:19
    at Generator.next (<anonymous>)
    at fulfilled (/Users/jspears/dev/schema-typed/node_modules/dtslint/bin/index.js:6:58)
```

Reply:
`dtslint` is a linting tool used by the current library before refactoring through typescript, I think it can be removed, because there is eslint as a type checking tool.